src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -67,8 +85,9 @@
       + "use the same "
       + "chroot path in its connection string. For example to give a chroot path of /chroot/path "
       + "you would give "
-      + "the connection string as hostname1:port1,hostname2:port2,hostname3:port3/chroot/path.";
-  public static final String ZOOKEEPER_CONNECT_DEFAULT = "localhost:2181";
+      + "the connection string as hostname1:port1,hostname2:port2,hostname3:port3/chroot/path. "
+      + "This is required to be configured only to use v1 Consumer API's.";
```

Comment:
Maybe 'NOTE: Only required when using v1 Consumer API's.' right at the beginning of the connection string. I think getting this message across is actually more important than what the property does at this point.